homeassistant/components/iaqualink/light.py

```diff
@@ -75,12 +76,12 @@ def brightness(self) -> int:
     @property
     def effect(self) -> str:
         """Return the current light effect if supported."""
-        return AqualinkLightEffect(self.dev.effect).name
+        return self.dev.effect
 
     @property
     def effect_list(self) -> list:
         """Return supported light effects."""
-        return list(AqualinkLightEffect.__members__)
+        return list(self.dev.supported_light_effects.keys())
```

Comment:
```suggestion
        return list(self.dev.supported_light_effects)
```

`.keys()` is the default when iterating